src/file-utils/edfi_lms_file_utils/constants.py

```diff
@@ -1,18 +1,43 @@
-# SPDX-License-Identifier: Apache-2.0
-# Licensed to the Ed-Fi Alliance under one or more agreements.
-# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-# See the LICENSE and NOTICES files in the project root for more information.
-
-ASSIGNMENT = "assignment"
-ASSIGNMENTS = "assignments"
-ATTENDANCE_EVENTS = "attendance-events"
-GRADES = "grades"
-LMS_SECTION_SOURCE_SYSTEM_IDENTIFIER = "LMSSectionSourceSystemIdentifier"
-SECTION = "section"
-SECTIONS = "sections"
-SECTION_ASSOCIATIONS = "section-associations"
-SECTION_ACTIVITIES = "section-activities"
-SOURCE_SYSTEM_IDENTIFIER = "SourceSystemIdentifier"
-SUBMISSIONS = "submissions"
-SYSTEM_ACTIVITIES = "system-activities"
-USERS = "users"
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# Resources
+class Resources:
+    ASSIGNMENT = "assignment"
+    ASSIGNMENTS = "assignments"
+    ATTENDANCE_EVENTS = "attendance-events"
+    GRADES = "grades"
+    SECTION = "section"
+    SECTIONS = "sections"
+    SECTION_ASSOCIATIONS = "section-associations"
+    SECTION_ACTIVITIES = "section-activities"
+    SUBMISSIONS = "submissions"
+    SYSTEM_ACTIVITIES = "system-activities"
+    USERS = "users"
+
+
+# Keys
+class Keys:
+    LMS_SECTION_SOURCE_SYSTEM_IDENTIFIER = "LMSSectionSourceSystemIdentifier"
+    SOURCE_SYSTEM_IDENTIFIER = "SourceSystemIdentifier"
+
+
+# Data types for dataframes
+
```

Comment:
I put in a lot of comments while this was in draft. Now that it is out of draft, I need one more comment in order to have an official review.
```suggestion
```